src/data/thunks.js

```diff
@@ -98,6 +97,32 @@ export function getProctoringSettings() {
   };
 }
 
+// todo: should this happen on its own or within updateAttemptAfter or somewhere else?
+// todo: need to update exam state to include exam access ticket for 1599 or later work?
+export function getExamAccessTicket() {
+  return async (dispatch, getState) => {
+    const { exam } = getState().examState;
+    if (!exam.id) {
+      logError('Failed to get exam settings. No exam id.');
+      handleAPIError(
+        { message: 'Failed to fetch exam access ticket. No exam id was found.' },
+        dispatch,
+      );
+      // return;
+    }
+    // try {
+    //   const examAccessTicketResponse = await fetchExamAccessTicket(exam.id);
+    //   const examAccessTicket = examAccessTicketResponse.cookies.exam_access_ticket;
+    //   // todo: error if access ticket is expired. Is it possible for it to exist if expired?
```

Comment:
My understanding was that the cookie would be deleted if expired, so I wasn't sure if this check was required. But it was something that came up as a potential AC.

Reply:
How does the cookie get deleted on expiration?

I know that the unpack function in token-utils, which the platform will call, will raise an error if the cookie is expired. Is that what you're referring to? Or is there some other process?

edx-exams will generate a new access token each time the access token API is hit, right? Do we plan on hitting that API only right before the learner enters the exam? In what cases is it probable that the token will have expired? I'd imagine it's very unlikely the token would have expired at this point, right?

Reply:
Ah, I just saw Zach's comment. This may be moot then.

Reply:
Can the backend return an expired access ticket? If yes, we would want to handle that case.